modules/project/README.md

```diff
@@ -317,9 +317,53 @@ module "service-project" {
 # tftest modules=2 resources=9 inventory=shared-vpc-auto-grants.yaml e2e
 ```
 
-In specific cases it might make sense to selectively grant the `compute.networkUser` role for service identities at the subnet level, and while that is best done via org policies it's also supported by this module.
```

Comment:
We should keep this sentence, as you're nuking the explanation for service identities. :)

Reply:
I just moved that sentence below the new example section